libs/devices/harmony/routes/src/lib/screens/manage-files/harmony-manage-files.config.tsx

```diff
@@ -45,26 +45,26 @@ export const HARMONY_SEGMENTS_CONFIG_MAP: Record<SegmentId, ISegmentBarItem> = {
 
 export const HARMONY_CATEGORIES_CONFIG_MAP: Record<
   FileCategoryId,
-  FileManagerFileCategory
+  FileManagerFileCategory & { id: FileCategoryId }
 > = {
   [FileCategoryId.alarmFiles]: {
     id: FileCategoryId.alarmFiles,
-    icon: IconType.Info,
+    icon: IconType.Bell,
     markerColor: FileManagerMarkerColor.alarmFiles,
     label: "Alarms",
     directoryPath: HarmonyDirectory.Alarm,
-    fileListEmptyStateDescription: "",
+    fileListEmptyStateDescription: "Add alarm files from your computer and they’ll transfer to your device automatically.",
     supportedFileTypes: ["mp3", "wav", "flac"],
     size: "0 KB",
     count: 0,
   },
   [FileCategoryId.relaxationFiles]: {
     id: FileCategoryId.relaxationFiles,
-    icon: IconType.Info,
+    icon: IconType.MuditaLogo,
     markerColor: FileManagerMarkerColor.relaxationFiles,
     label: "Relaxations",
     directoryPath: HarmonyDirectory.Relaxation,
-    fileListEmptyStateDescription: "",
+    fileListEmptyStateDescription: "Add relaxation files from your computer and they’ll transfer to your device automatically.",
```

Comment:
This should come from the translations file.

Reply:
b1f69ae4